src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/generated.py

```diff
@@ -54,11 +54,11 @@
 cli_command('webapp deployment user show', ProviderOperations.get_publishing_user, factory)
 
 factory = lambda _: web_client_factory().server_farms
-cli_command('webapp plan create', create_app_service_plan)
-cli_command('webapp plan update', update_app_service_plan)
-cli_command('webapp plan delete', ServerFarmsOperations.delete_server_farm, factory)
-cli_command('webapp plan list', ServerFarmsOperations.get_server_farms, factory)
-cli_command('webapp plan show', ServerFarmsOperations.get_server_farm, factory)
+cli_command('app-service-plan create', create_app_service_plan)
```

Comment:
/cc:@jasonrshaver @mayurid for suggestions on how the federation model works between us and service team when we have arguments.
You know, i have already tried not to avoid that, but service team did want it 
https://github.com/Azure/azure-xplat-cli/pull/3152/files#r78828301